### DISCUSSION TITLE
Java -> Kotlin = no world load

### DISCUSSION THREAD

**Discussion by zax71** [Minestom/Minestom#1601] [Help] [answered]
When converting my project to Kotlin, it seams to have gone well and is working correctly when run inside of IntelliJ but once compiled the world doesn't load but all the other code seams to run. I have no idea what info would be required so I can give it when asked 🙂

**Comment by Tofaa2** [comment]
Could you show your code? Both versions

**Reply by zax71** [reply]
New → [https://github.com/JavaDeathrun/lightHub](https://github.com/JavaDeathrun/lightHub)
Old → [https://github.com/JavaDeathrun/lightHub/tree/oldJava](https://github.com/JavaDeathrun/lightHub/tree/oldJava)

**Comment by zax71** [comment, marked as answer]
Emortal helped me out with this one on Discord and it ended up being because I used relative paths for the world folder. Once adding a config option for the world file location and actually having the world file in the correct location it works fine :)
